demos/mobile/android/app/src/main/java/com/google/blockly/android/webview/BlocklyWebViewFragment.java

```diff
@@ -0,0 +1,37 @@
+package com.google.blockly.android.webview;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+
+/**
+ * This fragments contains and manages the web view that hosts Blockly.
+ */
+public class BlocklyWebViewFragment extends Fragment {
+    protected @Nullable WebView mWebView = null;
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+            @Nullable Bundle savedInstanceState) {
+        mWebView = new WebView(inflater.getContext());
+        WebSettings webSettings = mWebView.getSettings();
+        webSettings.setJavaScriptEnabled(true);
+        mWebView.loadUrl("file:///android_asset/blockly/webview.html");
+        return mWebView;
+    }
+
+    // TODO: Method to invoke code generation
```

Comment:
Are these TODOs to you or to a developer using this example?

Reply:
Both, at the moment.